sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -202,7 +202,7 @@ def s3_event_handler(event):
     #     "extension": self.extension,
     #     "ingestion_date": date_object,
     # }
-    metadata_params = science_file.get_file_metadata_params()
+    metadata_params = science_file.extract_filename_components(filename)
```

Comment:
I can change that to be `file_params`.

Reply:
It should actually probably be `ScienceFilePath.extract_filename_components(filename)` and you remove the `science_file` variable entirely. Since you're just extracting the filename components, you don't need to use the rest of the class.

Reply:
I thought of doing that but was hesitant. will change to do that.